### ISSUE TITLE
function QO:IsParentQuestActive()  unused and not working

### ISSUE THREAD

**Issue by Laumesis** [Questie/Questie#3114]
Is this `function QO:IsParentQuestActive()` in master branch some left over or to support something in future? It is not used anywhere currently as I can see. And it refers to `self.parentID` which is **never defined**.

https://github.com/Questie/Questie/blob/f0da58bf24d91ef5c7401a275ecfb2421f42d501/Database/QuestieDB.lua#L704-L714

**Comment by BreakBB** [comment]
I guess a left over, not sure though. But if it is not needed, let's remove it.

Will you create a PR for it@Laumesis ?

**Comment by Laumesis** [comment]
There is also some other `parentID` related stuff. I wonder if these all are left overs?
```
fgrep -r parentID *
Database/QuestieDB.lua:---@param parentID number
Database/QuestieDB.lua:function QuestieDB:IsParentQuestActive(parentID)
Database/QuestieDB.lua:    if parentID == nil or parentID == 0 then
Database/QuestieDB.lua:    if QuestiePlayer.currentQuestlog[parentID] then
Database/QuestieDB.lua:        local parentID = self.parentID
Database/QuestieDB.lua:        if parentID == nil or parentID == 0 then
Database/QuestieDB.lua:        if QuestiePlayer.currentQuestlog[parentID] then
Libs/LibUIDropDownMenu/LibUIDropDownMenu.lua:                   listFrame.parentID = anchorFrame:GetID();
Modules/Quest/QuestieQuestPrivates.lua:function _QuestieQuest:IsParentQuestActive(parentID)
Modules/Quest/QuestieQuestPrivates.lua:    if parentID == nil or parentID == 0 then
Modules/Quest/QuestieQuestPrivates.lua:    if QuestiePlayer.currentQuestlog[parentID] then
```
```
fgrep -r IsParentQuestActive *
Database/QuestieDB.lua:function QuestieDB:IsParentQuestActive(parentID)
Database/QuestieDB.lua:        local isParentQuestActive = QuestieDB:IsParentQuestActive(parentQuest)
Database/QuestieDB.lua:    function QO:IsParentQuestActive()
Modules/Quest/QuestieQuestPrivates.lua:function _QuestieQuest:IsParentQuestActive(parentID)
```

And has it been same thing as nowadays `parentQuest` ? And might there be possible bugs somewhere using wrong one?
```
fgrep -r parentQuest *
Database/Classic/classicQuestDB.lua:    ['parentQuest'] = 25, -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 253,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 310,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 533,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 578,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 8554, -- #1691
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 924, -- #806
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 100, -- #1658
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 1119, -- #1084
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 1654,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 1654,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 2201,
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 3449, -- #1008
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 4784, -- #1367
Database/Corrections/Classic/classicQuestFixes.lua:            [questKeys.parentQuest] = 8258,
Database/questDB.lua:    ['parentQuest'] = 25, -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
Database/questDB.lua:    ['parentQuest'] = "u16", -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
Database/questDB.lua:    'requiredMinRep', 'requiredMaxRep', 'nextQuestInChain', 'questFlags', 'specialFlags', 'parentQuest',
Database/QuestieDB.lua:    local parentQuest = QuestieDB.QueryQuestSingle(questId, "parentQuest")
Database/QuestieDB.lua:    if parentQuest and parentQuest ~= 0 then
Database/QuestieDB.lua:        local isParentQuestActive = QuestieDB:IsParentQuestActive(parentQuest)
Database/QuestieDB.lua:    ---@field public parentQuest table
Database/TBC/tbcQuestDB.lua:    ['parentQuest'] = 25, -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
Modules/Journey/tabs/QuestsByZone/QuestsByZone.lua:                        "parentQuest",
Modules/Journey/tabs/QuestsByZone/QuestsByZone.lua:                local parentQuest = queryResult[3]
Modules/Journey/tabs/QuestsByZone/QuestsByZone.lua:                elseif parentQuest and Questie.db.char.complete[parentQuest] then
Modules/Libs/QuestieCompressedTable.lua:        --[[done]]['parentQuest'] = 25, -- int, the ID of the parent quest that needs to be active for the current one to be available. See also 'childQuests' (field 14)
Modules/Libs/QuestieCompressedTable.lua:        [questKeys['parentQuest']] = 13,
Modules/Libs/QuestieCompressedTable.lua:        stream:WriteShort(value[questKeys['parentQuest']] or 0)
Modules/QuestieEventHandler.lua:    if quest and ((quest.parentQuest and quest.IsRepeatable) or quest.Description == nil or table.getn(quest.Description) == 0) then
```

**Comment by Laumesis** [comment]
Can be closed, already fixed in #3509
